projects/online/online/main.py

```diff
@@ -169,21 +114,23 @@ def search(
                 )
                 if event is not None:
                     # maybe process event found in the previous frame
-                    process_event(
-                        event,
-                        gdb,
-                        input_buffer,
-                        spectral_density,
-                        pe_whitener,
-                        amplfi,
-                        scaler,
-                        pastro_model,
-                        samples_per_event,
-                        inference_params,
-                        outdir,
-                        nside,
-                        device,
+                    # event_processor(event)
+                    logging.info("Putting event in event queue")
+                    event_queue.put(event)
+                    logging.info("Running AMPLFI")
+                    descaled_samples = run_amplfi(
+                        event_time=event.gpstime,
+                        input_buffer=input_buffer,
+                        samples_per_event=samples_per_event,
+                        spectral_density=spectral_density,
+                        amplfi_whitener=amplfi_whitener,
+                        amplfi=amplfi,
+                        std_scaler=scaler,
+                        device=device,
                     )
+                    for i, sample in enumerate(descaled_samples.flatten()):
```

Comment:
Can you explain why this is needed here but not below at line 208

Reply:
Ah, this is leftover from when I thought I needed to update the shared memory this way. I changed it below but not here.